ann_visualizer/visualize.py

```diff
@@ -27,180 +27,186 @@ def ann_viz(model, view=True, filename="network.gv", title="My Neural Network"):
 
         title: A title for the graph
     """
-    from graphviz import Digraph;
-    import keras;
-    from keras.models import Sequential;
-    from keras.layers import Dense, Conv2D, MaxPooling2D, Dropout, Flatten;
-    import json;
-    input_layer = 0;
-    hidden_layers_nr = 0;
-    layer_types = [];
-    hidden_layers = [];
-    output_layer = 0;
+    from graphviz import Digraph
+    from tensorflow.python.keras.layers import Conv2D, MaxPooling2D, Activation, Dense
+    from tensorflow.python.keras.layers.core import Dropout, Flatten
+    # import keras
+    input_layer = 0
+    hidden_layers_nr = 0
+    layer_types = []
+    hidden_layers = []
+    output_layer = 0
     for layer in model.layers:
-        if(layer == model.layers[0]):
-            input_layer = int(str(layer.input_shape).split(",")[1][1:-1]);
-            hidden_layers_nr += 1;
-            if (type(layer) == keras.layers.core.Dense):
-                hidden_layers.append(int(str(layer.output_shape).split(",")[1][1:-1]));
-                layer_types.append("Dense");
+        if layer == model.layers[0]:
+            input_layer = int(str(layer.input_shape).split(",")[1][1:-1])
+            hidden_layers_nr += 1
+            if type(layer) == Dense:
```

Comment:
Perhaps, rewrite this as:
```suggestion
            if isinstance(Dense, layer):
```